docs/getting_started.rst

```diff
@@ -54,6 +54,83 @@ Connecting with a client
     # Disconnect from the server
     client.disconnect()
 
+**************
+Authentication
+**************
+
+When constructing a ``Cloudant`` client, you can authenticate using the
+[cookie authentication](http://guide.couchdb.org/editions/1/en/security.html#cookies) functionality.
+If you need the cookie to automatically renew when it expires,
+construct the client using ``auto_renew=True``:
+
+.. code-block:: python
+
+    # Create client using auto_renew to automatically renew expired cookie auth
+    client = Cloudant(USERNAME, PASSWORD, url='https://acct.cloudant.com',
+                     connect=True,
+                     auto_renew=True)
+
+****************
+Resource sharing
+****************
+
+The ``Cloudant`` or ``CouchDB`` client objects make HTTP calls using the ``requests`` library.
+``requests`` uses the [urllib3](https://pypi.python.org/pypi/urllib3) library which features
+connection pooling and thread safety.
+
+Connection pools can be managed by using the ``requests`` library's
+[HTTPAdapter](https://github.com/kennethreitz/requests/blob/master/requests/adapters.py#L78)
+when constructing a ``Cloudant`` or ``ClouchDB`` client instance.
+The default number set by the ``urllib3`` library for cached connection pools is 10.
+Use the ``HTTPAdapter`` argument ``pool_connections`` to set the number of
+urllib3 connection pools to cache, and the ``pool_maxsize`` argument to set the
+maximum number of connections to save in the pool.
+
+As long as the state of ``client`` does not changed it will be
```

Comment:
Will remove this with a [new comment about `client` per thread.](https://github.com/cloudant/python-cloudant/pull/272#discussion_r95386901)